### ISSUE TITLE
Please allow to use external zita-convolver

### ISSUE THREAD

**Issue by yurivict** [zamaudio/zam-plugins#69]
```USE_SYSTEM_LIBS``` disappeared, and ```HAVE_ZITA_CONVOLVER=true``` breaks:

```
gmake[2]: *** No rule to make target '../../build/ZamHeadX2/zita-convolver.cpp.o', needed by '../../bin/ZamHeadX2'.  Stop.
```

**Comment by jtojnar** [comment]
https://github.com/zamaudio/zam-plugins/pull/65#issuecomment-425210502

**Comment by zamaudio** [comment]
Fixed in 191d630
